src/router/Router.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import NavBar from "../components/NavBar";
+import Information from "../Pages/Information";
+import Profile from "../Pages/Profile";
+import FAQ from "../Pages/FAQ";
+import Registration from "../Pages/Registration";
+
+const Router = () => (
+  <BrowserRouter basename="/HacktoberFest">
+    <div className="App">
```

Comment:
Should we add NavBar to the className here? It would allow us to style the entire navbar across the screen like so:
```css
.NavBar {
  background-color: white;
  min-width: 100%;
}
```